synapse/models/economic.py

```diff
@@ -296,5 +339,125 @@ def getModelDefs(self):
+                        'doc': 'Set to true if the invoice has been paid in full.'}),
+
+                    ('balance', ('econ:price', {}), {
+                        'doc': 'The balance due.'}),
+
+                    ('currency', ('econ:currency', {}), {
+                        'doc': 'The currency that the invoice specifies for payment.'}),
+                )),
+
+                ('econ:acct:receipt', {}, (
+
+                    ('issued', ('time', {}), {
+                        'doc': 'The time the receipt was issued.'}),
+
+                    ('purchase', ('econ:purchase', {}), {
+                        'doc': 'The purchase that the receipt confirms payment for.'}),
+
+                    ('issuer', ('ps:contact', {}), {
+                        'doc': 'The contact information for the entity who issued the receipt.'}),
+
+                    ('recipient', ('ps:contact', {}), {
+                        'doc': 'The contact information for the entity who received the receipt.'}),
+
+                    ('currency', ('econ:currency', {}), {
+                        'doc': 'The currency that the receipt uses to specify the price.'}),
+
+                    ('amount', ('econ:price', {}), {
+                        'doc': 'The price that the receipt confirms was paid.'}),
+                )),
+
+                ('econ:bank:aba:rtn', {}, (
+
+                    ('bank', ('ou:org', {}}), {
+                        'doc': 'The bank which was issued the ABA RTN.'}),
+
+                    ('bank:name', ('ou:name', {}), {
+                        'doc': 'The name which is registered for this ABA RTN.'}),
+
+                )),
+
+                ('econ:bank:iban', {}, ()),
+
+                ('econ:bank:swift:bic', {}, (
+
+                    ('business', ('ou:org', {}), {
+                        'doc': 'The business which is the registered owner of the SWIFT BIC.'}),
+
+                    ('office', ('ps:contact', {}), {
+                        'doc': 'The branch or office which is specified in the last 3 digits of the SWIFT BIC.'}),
+                )),
+
+                ('econ:bank:account:type:taxonomy', {}, ()),
+                ('econ:bank:account', {}, (
+
+                    ('type', ('econ:bank:account:type:taxonomy', {}), {
+                        'doc': 'The type of bank account.'}),
+
+                    ('aba:rtn', ('econ:bank:aba:rtn', {}), {
+                        'doc': 'The ABA routing transit number for the bank which issued the account.'}),
+
+                    ('number', ('str', {'regex': '[0-9]+'}), {
+                        'doc': 'The account number.'}),
+
+                    ('iban', ('econ:bank:iban', {}), {
+                        'doc': 'The IBAN for the account.'}),
+
+                    ('owner', ('ps:contact', {}), {
+                        'doc': 'The contact information associated with the account.'}),
+
+                    ('issuer', ('ou:org', {}), {
```

Comment:
The :issuer property type for a receipt, invoice, etc is a ps:contact. Do we want this property to be consistent with the others? Do you want both an :issuer and :org prop here?

Reply:
Since a person can issue a receipt, the property must be a `ps:contact`